### ISSUE TITLE
No tasks loaded 

### ISSUE THREAD

**Issue by Trudko** [nickclaw/atom-grunt-runner#37]
Hi, maybe I am doing something wrong, but running the run command  opens empty list for me: 

![grunt-runner](https://cloud.githubusercontent.com/assets/3289225/6513670/e9dff768-c37c-11e4-8f98-cc98cfee4798.gif)

**Comment by kokarn** [comment]
Yeah this has been an issue for me as well since the last version. It still
works tho

**Comment by leshow** [comment]
I also can't get it to list more than one task at once per session, for example if i start up atom and go ctl+alt+shift+T and type in 'build'

the task will execute
if I then go
ctl+alt+g and type in 'default' (which runs a watch task)

it will not execute

**Comment by ealves-pt** [comment]
Any updates on this one?

**Comment by StormPooper** [comment]
I'm having the same issue, fresh install of Atom, default settings, `Gruntfile.js` is in the project root. If I type a task name in manually I get no output in the log and nothing appears to happen.

**Comment by kokarn** [comment]
Does it work from CLI? 
Can you post your Gruntfile?

**Comment by StormPooper** [comment]
@kokarn yes, it's used on our local machines and our build environments every day with no issues. It's quite large, so I've stuck it in a [PasteBin](http://pastebin.com/VxxtGraT).

**Comment by kokarn** [comment]
What plugins are you using for grunt? Maybe grunt-contrib-requirejs ?

**Comment by StormPooper** [comment]
No, we use bower components and wiredep at the moment, so no require stuff. Here's the [package.json](http://pastebin.com/DGqaFEvK).

**Comment by kokarn** [comment]
Thank you! I'll try to have a look at this but I'm a bit swamped so no promises :)

**Comment by StormPooper** [comment]
@kokarn no problem, thanks :)

**Comment by JohaWeber** [comment]
I'm having the same issue. Gruntfile works fine with Sublime Editor and a simliar plugin and also when directly started via CLI.

OS: Win 10, Atom: v.1.9.0-beta0. (Same issue with the stable realease of Atom 1.8.0)

**Comment by typhoon2099** [comment]
I might have the same issue. I click "Start Grunt" and no commands are listed. Nothing is written to the log. I can't tell if Grunt isn't running or failing to run or what's going on. It runs fine in a terminal.

**Comment by kokarn** [comment]
I have also got this error in one of my projects now. It includes nested folders in the project so might be that. Not sure tho.
